src/sass_types/null.cpp

```diff
@@ -15,45 +13,49 @@ namespace SassTypes
     return singleton_instance;
   }
 
-  Handle<Function> Null::get_constructor() {
+  v8::Local<v8::Function> Null::get_constructor() {
+    Nan::EscapableHandleScope scope;
+    v8::Local<v8::Function> conslocal;
     if (constructor.IsEmpty()) {
-      Local<FunctionTemplate> tpl = NanNew<FunctionTemplate>(New);
+      v8::Local<v8::FunctionTemplate> tpl = Nan::New<v8::FunctionTemplate>(New);
 
-      tpl->SetClassName(NanNew("SassNull"));
+      tpl->SetClassName(Nan::New("SassNull").ToLocalChecked());
       tpl->InstanceTemplate()->SetInternalFieldCount(1);
 
-      NanAssignPersistent(constructor, tpl->GetFunction());
+      constructor.Reset(Nan::GetFunction(tpl).ToLocalChecked());
+      conslocal = Nan::New(constructor);
 
-      NanAssignPersistent(get_singleton().js_object, NanNew(constructor)->NewInstance());
-      NanSetInternalFieldPointer(NanNew(get_singleton().js_object), 0, &get_singleton());
-      NanNew(constructor)->Set(NanNew("NULL"), NanNew(get_singleton().js_object));
+      get_singleton().js_object.Reset(conslocal->NewInstance());
+      Nan::SetInternalFieldPointer(Nan::New(get_singleton().js_object), 0, &get_singleton());
+      Nan::Set(conslocal, Nan::New("NULL").ToLocalChecked(), Nan::New(get_singleton().js_object));
 
       constructor_locked = true;
+    } else {
+      conslocal = Nan::New(constructor);
     }
 
-    return NanNew(constructor);
+    return scope.Escape(conslocal);
   }
 
   Sass_Value* Null::get_sass_value() {
     return sass_make_null();
   }
 
-  Local<Object> Null::get_js_object() {
-    return NanNew(this->js_object);
+  v8::Local<v8::Object> Null::get_js_object() {
+    return Nan::New(this->js_object);
```

Comment:
Sorry to be a PITA, I'm learning this way...

In the [current code](https://github.com/saper/node-sass/blob/a1395134907c6167a71487f5937869f7ca7d21b2/src/callback_bridge.h#L80-L91) we have only few `NewNan`s, I guess it works only because we are lucky:

````cpp
if (this->is_sync) {
    std::vector<Handle<Value>> argv_v8 = pre_process_args(argv);
    argv_v8.push_back(NanNew(wrapper));

    return this->post_process_return_value(
      NanNew<Value>(this->callback->Call(argv_v8.size(), &argv_v8[0]))
    );
  }
````

Thank you again!